cypress.json

```diff
@@ -5,5 +5,6 @@
   "env": {
     "privateKey": "7c6e602a94f30e4ea7edabe4376314f69ba7eaa2f355ecedb339df847b6f0d80575f81ffb0a297b7725dc671da0b1769b1fc5cbe45385c7b5ad1fc2eaf1d609d",
     "publicKey": "ak_fUq2NesPXcYZ1CcqBcGC3StpdnQw3iVxMA3YSeCNAwfN4myQk"
-  }
+  },
+  "chromeWebSecurity": false
```

Comment:
is it because of iframes?

Reply:
Jitsi requires to be running in `https` only

Reply:
Currently, it refers to Jitsi instance at league.aeternity.org, https should work fine there.

Reply:
it's 
```
To enable cross-domain iframe access, I will set the property chromeWebSecurity to false in the file cypress.json and re-run the test.
```
https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/